mixins/modalHelper.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Helpers for modals
+ *
+ * @mixin
+ */
+import Vue from 'vue';
+
+declare module '@nuxt/types' {
+  interface Context {
+    errorModalWidth: () => string | number;
+  }
+
+  interface NuxtAppOptions {
+    errorModalWidth: () => string | number;
+  }
+}
+
+declare module 'vue/types/vue' {
+  interface Vue {
+    errorModalWidth: () => string | number;
+  }
+}
+
+const modalHelper = Vue.extend({
+  computed: {
+    errorModalWidth(): string | number {
+      if (this.$vuetify.breakpoint.smAndDown) {
```

Comment:
I don't understand very well this condition. Why so many else if? Can't we just do this with a simple if-else? We're returning 600 everywhere.

Reply:
We can simplify it, for sure. I mainly left it as-is in case we wanted different sizes for the errorModal at some point. I also think that "errorModalWidth" shouldn't be the name of this... maybe like "smallModalWidth" ? Make it more generic.

Reply:
@MrTimscampi 100% agree with the name change.

As for the condition, I would leave it as a simple if-else. Revisiting this later shouldn't take so much effort, so I don't really see a reason to keep it for now.

I won't block this if you really want to keep this, but please add a comment stating the reasons, so they're clear in the future.